py/persistentcode.c

```diff
@@ -377,7 +377,7 @@ void mp_raw_code_save(mp_raw_code_t *rc, mp_print_t *print) {
 // here we define mp_raw_code_save_file depending on the port
 // TODO abstract this away properly
 
-#if defined(__i386__) || defined(__x86_64__) || defined(__unix__)
+#if defined(__i386__) || defined(__x86_64__) || defined(_WIN32) || defined(__unix__)
```

Comment:
Does this need a `_WIN64` as well?

Reply:
No, despite the `32` there, _WIN32 is defined as 1 when the compilation target is 32-bit ARM, 64-bit ARM, x86, or x64